website/docs/d/ecs_cluster.html.markdown

```diff
@@ -34,4 +34,6 @@ In addition to all arguments above, the following attributes are exported:
 * `pending_tasks_count` - The number of pending tasks for the ECS Cluster
 * `running_tasks_count` - The number of running tasks for the ECS Cluster
 * `registered_container_instances_count` - The number of registered container instances for the ECS Cluster
+* `capacity_providers` - The capacity providers registered with the ECS Cluster.
```

Comment:
Updates to the data source are not planned for the initial release, so we should leave this file alone for now. 👍